pilot/docker/wait-ready.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
```

Comment:
This won't work on the distroless images. I think thats fine for now though

Reply:
Yeah, this whole script should really be replaced with go code. I wanted to check first if this whole approach is even acceptable before spending too much time on implementation.

Reply:
I would think we merge this in as is (assuming no one else has objection, LGTM but I don't want to merge this 20 minutes after creation without others taking a look) and get some feedback then we can refine it

Reply:
Absolutely. Also, if anyone has any comments on the name/prefix of the annotation, I'd love to hear them.